doc/md/ddl/sql_types.md

```diff
@@ -187,6 +187,7 @@ slug: /ddl/sql-types
             <td>decimal</td>
             <td>
                 <ul>
+                        <li>unsigned (bool)</li>
```

Comment:
Not part of my PR, but CI is falling.

Reply:
Shouldn't we fix it first then and rebase this PR onto that?

Reply:
I don't mind landing this PR with this failure. It's not a test issue.

Reply:
Fixed.